src/main/java/com/lorepo/icplayer/client/module/skiplink/SkipLinkModule.java

```diff
@@ -0,0 +1,165 @@
+package com.lorepo.icplayer.client.module.skiplink;
+
+import com.google.gwt.xml.client.Element;
+import com.google.gwt.xml.client.Node;
+import com.google.gwt.xml.client.NodeList;
+import com.lorepo.icf.properties.IListProperty;
+import com.lorepo.icf.properties.IPropertyProvider;
+import com.lorepo.icf.utils.XMLUtils;
+import com.lorepo.icf.utils.i18n.DictionaryWrapper;
+import com.lorepo.icplayer.client.module.BasicModuleModel;
+import com.lorepo.icplayer.client.module.skiplink.interfaces.ISkipLinkItem;
+import com.lorepo.icplayer.client.module.skiplink.interfaces.ISkipLinkModule;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SkipLinkModule extends BasicModuleModel implements ISkipLinkModule {
+    private final List<SkipLinkItem> items;
+
+    public SkipLinkModule() {
+        super("SkipLink", DictionaryWrapper.get("skiplink_module"));
+        this.items = new ArrayList<SkipLinkItem>();
+
+        addPropertyItems();
+    }
+
+    @Override
+    public String getName() {
+        return "SkipLink";
+    }
+
+    @Override
+    public List<? extends ISkipLinkItem> getItems() {
+        return items;
+    }
+
+    @Override
+    public String toXML() {
+        Element module = XMLUtils.createElement("skipLinkModule");
+
+        setBaseXMLAttributes(module);
+        module.appendChild(getLayoutsXML());
+        module.appendChild(getXMLItems());
+
+        return module.toString();
+    }
+
+    @Override
+    protected void parseModuleNode(Element element) {
+        items.clear();
+
+        NodeList itemsTags = element.getElementsByTagName("items");
+        if (itemsTags.getLength() == 1) {
+            Node itemTag = itemsTags.item(0);
+
+            NodeList itemsList = itemTag.getChildNodes();
+            for (int i = 0; i < itemsList.getLength(); i++) {
```

Comment:
Można dać pętlę for (Obj obj: itemList) i pozbyć się itemsList.item(i).

Reply:
Tutaj niestety NodeList nie implementuje typowej kolekcji - trzeba pobierać za pomocą item.